data/languages/fr60.sinc

```diff
@@ -233,17 +233,17 @@ DIR8B_REL: reloc is dir8 [ reloc = dir8 * 1; ] {
 }
 
 :AND rj, @ri is op8=0x84 & rj & ri {
-    *ri = *:4 ri & rj;
+    *:4 ri = *:4 ri & rj;
     resultFlags(*:4 ri);
 }
 
 :ANDH rj, @ri is op8=0x85 & rj & ri {
-    *ri = *:2 ri & rj:2;
+    *:2 ri = *:2 ri & rj:2;
```

Comment:
thank you, I made the assumption that the target pointer would auto-size to the value being written to it. This cleans up a lot of the auto-type detecting

Reply:
I did not check the compiled pcode, but better safe than sorry I guess